### ISSUE TITLE
Why not include xo/config/plugins.js?

### ISSUE THREAD

**Issue by teehemkay** [sindresorhus/eslint-config-xo#30]
Unless I'm mistaken, it seems to me that the settings under `xo./config/plugins.js` are not included here.

E.g. all the rules related to the eslint-plugin-import` plugin.

Why is that, I would thought that these rules are quite useful?

I'm actually using `XO` but in some case (lack of editor integration) I need to revert to using `ESLint` directly with  'eslint-plugin-xo`,`eslint-config-xo` etc.

**Comment by sindresorhus** [comment]
It's because of https://github.com/eslint/eslint/issues/3458 (follow and comment on that thread). Shareable configs can't have dependencies, so I would have to force the user to manually install all the plugins XO uses.

**Comment by sindresorhus** [comment]
What editor integrations are you missing?

**Comment by teehemkay** [comment]
Thank you for the quick answer!

Re: the editor: WebStorm

**Comment by sindresorhus** [comment]
@teehemkay Re WebStorm, see: https://github.com/sindresorhus/xo/issues/67#issuecomment-227795061 Should be possible to set it up with XO.

**Comment by teehemkay** [comment]
Thank you for the pointer. I think it boils down to manually replicating all the ESLint setup / configuration that XO does automatically and adjusting WebStorm default settings so that they don't conflict with XO's...  

Which I had done already. I used `eslint-plugin-xo`, `eslint-config-xo-space` and `eslint-config-xo-react`) and created an `.eslintrc*`  config that's essentially `xo/config/plugins.js` extended with the `eslint-*-xo*` shareable configs above. 

It's was not complicated since thanks to you all the pieces are already readily available and I just needed to assemble them.

While writing this I had the an idea that maybe could help to automate the above.

But I see that someone else already suggested it :-): https://github.com/sindresorhus/xo/issues/76

So I'll add a comment there.

Thank you again for your help, I really appreciate it.
